tutorial/Linux_Tutorial.md

```diff
@@ -376,6 +376,16 @@ The current rosbag2 implementation provides plugins which allow to replay - and
 Needless to say, this requires a parallel ros1 melodic installation and thus only works on Linux 18.04.
 The plugin can be found here: https://github.com/ros2/rosbag2_bag_v2
 
+In a fresh terminal:
+```bash
+source /opt/ros/melodic/setup.bash
+rosbag info ros1_cmd_vel.bag
+```
+You can see that this is a ROS1 rosbag as usual.
+When specifying the plugin for reading legacy rosbags in ROS2, we can load the same information in ROS:
+```
+ros2 bag info -s rosbag_v2 ros1_cmd_vel.bag
+```
```

Comment:
That's nice! :+1:

Some magic happening there :open_mouth: 
```
$ rosbag info ros1_cmd_vel.bag | grep duration
duration:    21.1s
$ ros2 bag info -s rosbag_v2 ros1_cmd_vel.bag | grep Duration
Duration:          21.99s
```

Reply:
![magic](https://media.giphy.com/media/VHngktboAlxHW/giphy.gif)

Reply:
![mgc](https://user-images.githubusercontent.com/6644208/59074799-725ecf00-88cd-11e9-87e2-bac745bf3d93.gif)